bindings/dotnet/openDAQ.Net/Thoughts-about-CSharp-Bindings.txt

```diff
@@ -0,0 +1,193 @@
+Thoughts about C# Bindings
```

Comment:
I'm not sure if this file should be committed in the repo. Particularly with strange names like "Dejan"

Reply:
lol, you're right, will remove it from repo soon

Reply:
done (force push that it is also removed from history)